ibm/resource_ibm_cos_bucket.go

```diff
@@ -227,7 +260,7 @@ func resourceIBMCOSBucket() *schema.Resource {
 			"expire_rule": {
 				Type:        schema.TypeList,
 				Optional:    true,
-				MaxItems:    1000,
+				MaxItems:    1,
```

Comment:
you can create multiple expiry rules?
why the number is changed from 1000 to 1

Reply:
Max item should be 1 only . Once we change to 1000, it failed with below error in testing
```
resource_ibm_cos_bucket_test.go:92: Step 1/2 error: Error running pre-apply refresh: exit status 1
        2021/11/24 20:46:00 [DEBUG] Using modified User-Agent: Terraform/0.12.28 HashiCorp-terraform-exec/0.14.0

        Error: InternalValidate

        Internal validation of the provider failed! This is always a bug
        with the provider itself, and not a user issue. Please report
        this bug:

        1 error occurred:
        	* resource ibm_cos_bucket: ConflictsWith: expired_object_delete_marker
        configuration block reference (expire_rule.0.date) can only be used with
        TypeList and MaxItems: 1 configuration blocks
```